src/status.js

```diff
@@ -0,0 +1,137 @@
+import { checkBox, clearCont } from './check.js';
+
+const addTask = document.querySelector('.subheading');
```

Comment:
- [ ] Kindly make sure to clear the add task input field after successfully adding a task so users can add new tasks without clearing the field manually each time.

Reply:
Thank you, @omar-labana, for pointing this out! I had been meaning to implement that but totally forgot about it in the end. The input field now clears once the user types a task and hits enter. 👌